.github/CHANGELOG.md

```diff
@@ -1,38 +1,60 @@
-## Release 0.7.0a4 (development release)
+## Release 0.8.0a4 (development release)
+
+### New features since the last release
+
+
+### Bug fixes
+
+
+### Improvements
+
+
+### Documentation changes
+
+
+### Contributors
+
+This release contains contributions from (in alphabetical order):
+
+See full commit details ...
+
+---
+
+## Release 0.8.0a4 (current release)
 
 ### New features since the last release
 
 * Node and edge coloring can now be done based on any attribute and personalized colors can be defined via a dictionary: [#32](https://github.com/XanaduAI/flamingpy/pull/32)(backward compatible)
```

Comment:
```suggestion
* Node and edge coloring can now be done based on any attribute and personalized colors can be defined via a dictionary: [#32](https://github.com/XanaduAI/flamingpy/pull/32) (backward compatible)
```
I think backward incompatible. Drawing will no longer work if you set color_nodes to, say, "state". Now you have to supply a tuple(str, dict).